float64[] j_value
float64[] j_velocity
float64[] j_accel
